2996_reflection/reflection.md

```diff
@@ -3477,49 +3547,67 @@ Add a new paragraph between [expr.eq]{.sref}/5 and /6:
 
 ### [expr.const]{.sref} Constant Expressions {-}
 
-Add new paragraphs prior to the definition of _manifestly constant evaluated_ ([expr.const]{.sref}/20), and renumber accordingly:
+Modify paragraph 15 to disallow returning non-consteval-only pointers and references to consteval-only objects from constant expressions.
 
 ::: std
-::: addu
+[15]{.pnum} A _constant-expression_ is either a glvalue core constant expression that [\ ]{.addu}
 
-[20]{.pnum} An expression or conversion is _in substitution context_ if it results from the substitution of template parameters
+* [#.#]{.pnum} refers to an entity that is a permitted result of a constant expression[, and]{.addu}
+* [[#.#]{.pnum} is of consteval-only type if the entity designated by the expression is of consteval-only type,]{.addu}
```

Comment:
Add a note that this prohibits forming a `Base&` to a `Derived` if the latter is consteval-only but the former is not.

Reply:
Good idea, done.